env/src/main/java/com/sap/cloud/security/config/k8s/K8sConstants.java

```diff
@@ -0,0 +1,34 @@
+/*
+  SPDX-FileCopyrightText: 2018-2021 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
+  SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.cloud.security.config.k8s;
+
+import com.sap.cloud.security.config.cf.CFConstants;
+
+/**
+ * Constants that simplifies access to service configuration properties in the
+ * Kubernetes environment.
+ */
+public class K8sConstants {
```

Comment:
We can not remove classes from java-api, only from java-security, as java-security depends on env.

Reply:
it looked very tempting to move it there 🤣